resources/styles/components/questions-dashboard.less

```diff
@@ -23,6 +23,11 @@
     }
   }
 
+  .exercises-display {
+    max-width: @tutor-max-panel-width;
+    margin: 0 auto;
+  }
+
```

Comment:
We discussed this in the story and Kathi gave Robert the goahead on this design change.